module-2/backend/week_one.md

```diff
@@ -5,30 +5,80 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
-2. What is Sinatra?
+  POST - create
+  GET - read
+  PUT - update/replace
+  PATCH - update/modify
+  DELETE - delete
+  
+2. What is Sinatra? 
+  a framework for Ruby
+
 4. What is MVC?
+  the common folder structure for web applications consisting of model-the info tracker/where 'work' takes place, views-what users see, controller-the delegater/sends you to the correct location
+  
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+  to make our code easy to read/follow for other developers
+  
 6. What types of variables are accessible in our view templates without explicitly passing them?
+  HTML elements; all variables/data are sent from the controller

```

Comment:
Answer we were looking for is *instance variables* consistent with your answer to the next question.